### DISCUSSION TITLE
Hierarchical DEF

### DISCUSSION THREAD

**Discussion by kareefardi** [The-OpenROAD-Project/OpenROAD#3530] [Q&A]
I recall seeing commit relating to Hierarchical DEF views. Is it something currently, or planned to be, supported in OpenROAD now?

**Comment by oharboe** [comment]
> I recall seeing commit relating to Hierarchical DEF views. Is it something currently, or planned to be, supported in OpenROAD now?

Like this?

https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/issues/1176#issuecomment-1605301917
